src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Abstractions/IAirship.cs

```diff
@@ -53,5 +53,7 @@ int MessageCenterCount
         Channel.TagGroupsEditor EditChannelTagGroups();
         
         Attributes.AttributeEditor EditAttributes();
+
+        void RegisterEventListener(Action<string, Dictionary<string, string>> onEventReceived);
```

Comment:
C# has an event listener pattern that we should follow:

```
        public delegate void DeepLinkHandler (string deepLink);

        public event DeepLinkHandler OnDeepLink;
``` 

Then you add/remove using:
```
UAirship.instance.OnDeepLink += OnDeepLink;
UAirship.instance.OnDeepLink -= OnDeepLink;
```

Reply:
Noice, updated the listener.